package.json

```diff
@@ -19,41 +19,32 @@
     "node": ">=10"
   },
   "scripts": {
-    "build": "buidler compile",
-    "test": "buidler test test/*.ts",
-    "ganache": "./scripts/ganache.sh",
-    "deploy:ganache": "yarn ganache && truffle migrate",
-    "mainnet-ganache": "./scripts/mainnet-ganache.sh",
-    "coverage": "buidler coverage --network coverage --temp build --testfiles 'test/*.ts'",
+    "build": "hardhat compile",
+    "test": "hardhat test",
+    "coverage": "hardhat coverage",
     "lint": "prettier ./test/**/*.ts --check",
     "lint:ts": "prettier ./test/**/*.ts --write",
     "lint:sol": "solhint -f table contracts/**/*.sol"
   },
-  "dependencies": {
-    "@openzeppelin/contracts": "^3.1.0"
```

Comment:
ERC20Permit inherits from the OZ's ERC20.sol. If we remove this, users of the npm package will have to install openzeppelin-contracts themselves, right?

Reply:
Yes